module/User/view/user/email/register.phtml

```diff
@@ -1,16 +1,33 @@
-<?= sprintf($this->translate('Dear %s,'), $member->getFullName()) . "\n" ?>
+<?php $lang = 'en'; ?>
+For English, please see below.<br/><br/>
 
-<?= $this->translate('You have been registered on the website of Study association GEWIS, with the following information:') . "\n" ?>
 
-- <?= $this->translate('Membership number:') ?> <?= $user->getLidnr() . "\n" ?>
-- <?= $this->translate('Email address:') ?> <?= $user->getEmail() . "\n" ?>
+<?= sprintf($this->translate('Beste %s,'), $member->getFullName()) ?><br/><br/>
```

Comment:
Is it not easier to remove this `translate()` as well (and the seemingly unused `$lang`)?